.github/pull_request_template.md

```diff
@@ -1,8 +1,8 @@
 Checklist for reviewer:
 
-- [ ] Commits should reference a bug or github issue, if relevant
+- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
 - [ ] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
-- [ ] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional) and review the report posted in the comments.
+- [ ] Watch the results of the `integration` CI test and make sure the error rates do not increase without justification.
```

Comment:
The integration tests may still need to be triggered manually if the PR comes from a fork of the repository.

Reply:
I'll revert this change and make a note about the fork part.